pkg/core/auth/role/mock_store.go

```diff
@@ -74,6 +74,11 @@ func (m *MockStore) SetAdminRoles(roles []string) error {
 	return m.setRoleType(roles, "is_admin")
 }
 
+// SetDefaultRoles set role type to true
+func (m *MockStore) SetDefaultRoles(roles []string) error {
+	return m.setRoleType(roles, "is_default")
```

Comment:
To align with `pg/store.go`, let's use `by_default` instead of `is_default` (`setRoleType` needs to update too). Thanks.